web/themes/joinup/prototype/sass/components/_dropdown-menu.sass

```diff
@@ -0,0 +1,24 @@
+// Dropdown menu
+//
+// Dropdown component
+
+// Component
+// --------------------------------------------------
+//.dropdown-menu
```

Comment:
if this is the part where you make something reusable for ISAICP-3578 - isn't it worth making the html as well? (for the prototype)
Honestly, when I see this specific styling in a component file, I expect to have a bigger picture what it is for, if I am to reuse it. For now, the prototype shows a dropdown in the circle menu which is a component namespace and implementation by itself.